.github/workflows/javascript.yml

```diff
@@ -4,17 +4,17 @@ name: javascript
 on: [push, pull_request]
 jobs:
   test:
-    name: Node ${{ matrix.node-version }} and ${{ matrix.os }}
+    name: Lint JavaScript code (using Node version ${{ matrix.node-version }})
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        node-version: [16.x]
+        node-version: [18]
```

Comment:
This is 18, not 20 😉

Reply:
Oh, I forgot to actually stage that change in the end. Let's see whether 20 works as well because I suppose the newer the better.

By the way, I would have tried to avoid specifying a version at all but it is recommended by https://github.com/actions/setup-node.